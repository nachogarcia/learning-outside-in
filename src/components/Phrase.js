import React from 'react'

export default ({ joke }) => <div className='phrase'>
  <span className='joke'>{joke}</span>
  <hr/>
</div>
