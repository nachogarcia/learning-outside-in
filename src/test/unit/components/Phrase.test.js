import React from 'react'
import Phrase from 'src/components/Phrase'

const joke = 'some joke'

describe('Phrase', () => {
  let wrapper

  beforeEach(() => {
    wrapper = shallow(<Phrase joke={joke}/>)
  })

  it('displays joke', () => {
    expect(wrapper.contains(joke)).toBe(true)
  })

  it('has correct structure', () => {
    expect(wrapper.html()).toMatchSnapshot()
  })
})