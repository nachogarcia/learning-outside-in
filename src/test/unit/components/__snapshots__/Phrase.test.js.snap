// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Phrase has correct structure 1`] = `

<div class="phrase">
  <span class="joke">
    some joke
  </span>
  <hr>
</div>

`;
